embed_grader/serapis/templates/serapis/view_task_input.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'serapis/base_sidebar.html' %}
+{% load staticfiles %}
+{% load guardian_tags %}
+{% load submission_tags %}
+
+{% block profile-usertitle-name %}
+  {{ course.course_code }}
+{% endblock %}
+
+{% block profile-usertitle-job %}
+  {{ assignment.name }}
+{% endblock %}
+
+{% block courses-active %}active{% endblock %}
+
+{% block main-content %}
+  {% get_obj_perms request.user for course as "course_perms" %}
+
+  <div class="main-content">
+    <div class="top-buffer">
+      <ol class="breadcrumb">
+        <li><a href="{% url 'homepage' %}">Courses</a></li>
+        <li><a href="{% url 'course' course.id %}">{{course.course_code}}</a></li>
+        <li><a href="{% url 'assignment' assignment.id %}">{{assignment.name}}</a></li>
+        <li class="active">{{task.brief_description}}</a></li>
+      </ol>
+    </div>
+
+    <h2 class="page-header">Task - {{task.brief_description}} Input</h2>
```

Comment:
Can we follow the format of task-grading-detail page:

`<h3 class="page-header">Input files</h3>`

And then provide the following lines:

- Task name
- Mode
- Points
- Description

https://github.com/nesl/AutoGrader/blob/master/embed_grader/serapis/templates/serapis/task_grading_detail.html#L40